x-pack/legacy/plugins/maps/public/layers/styles/vector/properties/dynamic_style_property.js

```diff
@@ -99,9 +99,7 @@ export class DynamicStyleProperty extends AbstractStyleProperty {
   async getFieldMetaRequest() {
     if (this.isOrdinal()) {
       const fieldMetaOptions = this.getFieldMetaOptions();
-      return this._field.getOrdinalFieldMetaRequest({
```

Comment:
sigma was not being used in the request. The request just returns standard deviation.